sos/report/plugins/seagate_ses.py

```diff
@@ -51,6 +52,9 @@ def setup(self):
             'getvpd',
             'report_faults',
             'ver',
+            'logdump',
+            'phydump',
+            'ses_reportpage 2'
```

Comment:
minor nit, can we add a comma at the end please

Reply:
assuming we are getting ses page 2, "enclosure control / enclosure status", is there any reason why you don't want to capture  page 1 - Configuration as well?

Reply:
> minor nit, can we add a comma at the end please

Done.

Reply:
> assuming we are getting ses page 2, "enclosure control / enclosure status", is there any reason why you don't want to capture page 1 - Configuration as well?

SES page1 is configuration page and only defines the supported elements and their number (I mean number of supported elements)...not so significant page, therefore its not required.